docs/collections/_policies/json-format.md

```diff
@@ -599,7 +599,8 @@ JSON representation
 
 ### `annotations`
 
-Annotations, if present, must be a JSON object.  The keys and values, which must all be strings, correspond to the Cedar annotation keys and values on the policy.
+Annotations, if present, must be a JSON object.  The keys which must be strings and values which may be strings or the JSON value `null`, correspond to the Cedar annotation keys and values on the policy.
+A `null` value corresponds to an annotation without a value (`@shadow_mode`) and is equivalent to the value to `""`.
```

Comment:
Small typo in the last sentence. Also deleted part of the prior sentence because I found the phrasing confusing (although not technically wrong).

```suggestion
Annotations, if present, must be a JSON object.  The keys must be strings and the values may be strings or the JSON value `null`.
A `null` value corresponds to an annotation without a value (e.g., `@shadow_mode`) and is equivalent to the value `""`.
```